Sources/StatsdClient/StatsdClient.swift

```diff
@@ -278,20 +279,11 @@ private final class Client {
     }
 
     private func connect() -> EventLoopFuture<Channel> {
-        if let channel = self.channel.load().value {
-            return self.eventLoopGroup.next().makeSucceededFuture(channel)
-        }
-
         let bootstrap = DatagramBootstrap(group: self.eventLoopGroup)
             .channelOption(ChannelOptions.socket(SocketOptionLevel(SOL_SOCKET), SO_REUSEADDR), value: 1)
             .channelInitializer { channel in channel.pipeline.addHandler(Encoder(address: self.address)) }
-
         // the bind address is local and does not really matter, the remote address is addressed by AddressedEnvelope below
-        let future = bootstrap.bind(host: "0.0.0.0", port: 0)
-        future.whenSuccess { channel in
-            self.channel.store(Box(channel))
-        }
-        return future
```

Comment:
removed this channel caching. could also implement a state machine instead if we think its worth while (performance)